src/App.tsx

```diff
@@ -2,14 +2,29 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
-
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
-  </div>
-);
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { IPost } from './types/IPost';
+import { PostList } from './components/PostList/PostList';
+
+const preparedPosts: IPost[] = posts.map((post) => {
+  return {
+    ...post,
+    user: users.find(user => user.id === post.userId) || null,
+    comments: comments.filter(coment => coment.postId === post.id) || null,
```

Comment:
Empty array will return true when converted to boolean, so your comments field will never be null, fix it in your interface also
```suggestion
    comments: comments.filter(coment => coment.postId === post.id),
```

Reply:
fixed